### ISSUE TITLE
Plugin name incorrectly logged

### ISSUE THREAD

**Issue by NickDub** [TheUnderScorer/nx-semantic-release#98]
When I launch a release, I don't see the name of the plugin that will be executed for the following steps (in red): 
- analyzeCommits
- generateNotes
- prepare
- success

But it's okay for the other steps (in green).
![image](https://github.com/TheUnderScorer/nx-semantic-release/assets/20282929/490669c3-aaa0-445f-83d1-2f135271d9ec)
![image](https://github.com/TheUnderScorer/nx-semantic-release/assets/20282929/268997a2-0306-48ba-862f-01f531d88bdd)

Is it possible to have the plugin name and not `"[Function: nx-semantic-release]"`?

**Comment by TheUnderScorer** [comment]
Hey @NickDub! 

It's probably related to `semantic-release-plugin-decorators` plugin. I'm not quite sure how to fix it, perhaps future planed update to it will fix it.

**Comment by NickDub** [comment]
Hi @TheUnderScorer 

When do you plan to update to the latest version of `semantic-release-plugin-decorators` ?

**Comment by spencer-shupe-qhr** [comment]
Tested the 3.02 and 3.03 versions of `semantic-release-plugin-decorators` and the issue is not fixed, unfortunately. The 4.0.0 is of course a breaking change with ESM support, uncertain if that will fix it.

FWIW I stepped through with the debugger a little and found it strange that the `plugin` variable is resolved as a function reference with `name: 'nx-semantic-release'` instead of an array or object as expected here https://github.com/semantic-release/semantic-release/blob/master/lib/get-config.js#L44 (I traced started from https://github.com/semantic-release/semantic-release/blob/master/lib/plugins/normalize.js#L35 where `pluginName` is also a function reference with name: `'nx-semantic-release'`, though this could be a red-herring).
